src/core_plugins/metric_vis/public/components/metric_vis_value.js

```diff
@@ -26,6 +26,11 @@ class MetricVisValue extends Component {
       'metric-container--filterable': hasFilter
     });
 
+    /*
+     * Justification for dangerouslySetInnerHTML:
+     * This is one of the visualizations which makes use of the HTML field formatters.
+     * Since these formatters produce raw HTML, this visualization needs to be able to render them as-is.
```

Comment:
I suggest the following slight changes:

```
This is one of the visualizations which makes use of the HTML field formatters.
Since these formatters produce raw HTML, this visualization needs to be able to render them as-is, relying
on the field formatter to only produce safe HTML.
`metric.value` is set by the MetricVisComponent, so this component must make sure this value never contains
any unsafe HTML (e.g. by bypassing the field formatter).
```